app/assets/stylesheets/application.tailwind.css

```diff
@@ -1,4 +1,5 @@
 @import "./colors.tailwind.css";
+@import "./marketing.tailwind.css";
```

Comment:
did this to make sure ALL the tailwind is getting compiled due to the changes in #1